src/Web/E2E/playwright.config.ts

```diff
@@ -30,7 +30,7 @@ const config: PlaywrightTestConfig = {
     /* Base URL to use in actions like `await page.goto('/')`. */
     baseURL: process.env.BASEURL || 'https://dotnetpodcasts.azurewebsites.net',
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: 'on',
+    trace: 'on-first-retry',
```

Comment:
then I will add a comment saying we don't recommend doing this as otherwise people will copy this repo as something that you should do and we recommend against it and instead you should use the cli command `npx playwright test --trace on` when you want the trace locally. We actually built this command to stop people form putting trace: 'on' in the config as this was something I was previously showing off in conference talks as well and had to correct.

Reply:
Yeah, I get the pros and cons. Having to re-run passing `--trace` defeats the purpose of using this as a post-mortem debugging tool. The downside of taking a bit longer to run is a tradeoff worth taking IMO.